test/__fixtures__/substrate-node-api-fn.js

```diff
@@ -0,0 +1,72 @@
+module.exports = {
```

Comment:
This is a nice fixture. This is how I should have done this in the `dscp-api`

Reply:
Ideally would lik eto stick to the json's where possible, but this needed some prototype which could have been added here rather than in test.js fle e.g. `.toNumber()`